unleash-yggdrasil/src/strategy_grammar.pest

```diff
@@ -74,6 +75,9 @@ constraint = {
     )
 }
     default_strategy_constraint = { boolean }
+    hostname_constraint = { hostname ~ in ~ string_list }
+        hostname = _{ "hostname" }
```

Comment:
The little underscore here is a non emitting rule - it doesn't generate a node to iterate over in the parser. Chosen this because we don't care about anything outside of the contents of the list